.eslintrc.json

```diff
@@ -40,211 +37,45 @@
         ]
       }
     ],
+    "sort-keys": "off",
+    "arrow-body-style": "off",
```

Comment:
Let's sort these rules alphabetically for better readability.

Reply:
Done.